internal/server/data/provideruser.go

```diff
@@ -259,6 +259,7 @@ func GetProviderUser(tx ReadTxn, providerID, identityID uid.ID) (*models.Provide
 	query.B(columnsForSelect(pu))
 	query.B("FROM")
 	query.B(pu.Table())
+	query.B("LEFT OUTER JOIN providers ON provider_users.provider_id = providers.id AND providers.organization_id = ?", tx.OrganizationID())
```

Comment:
Is this related to the fix, or was this about something else?

Reply:
Should have called this one out, it was not related to the fix. Just noticed the inconsistency here between `GetProviderUser` and `ListProviderUser`. In `ListProviderUser` we do this join as a safety thing to scope the query to the current organization. We should do it in `GetProviderUser` too just in case.